### PR TITLE
Add Contact form

### PR DESCRIPTION
**Pull request by Lucas-Erkana**
-Add Name field that accepts up to 30 chars.
-Add Email field that accepts only emails.
-Add Message field that accepts up to 500 chars.
-Add placeholder text to all fields.
-Add fields that have the correct type.
- Make sure user cannot submit the form if one or more of the fields is empty.
- Collect data by using the [Formspree service](https://formspree.io/html).

### CODE REVIEW

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 

Hi @Lucash2022 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉🎉

approvedstamp

## Highlights 💯 
- No Linter errors ✔️ 
- Descriptive PR ✔️ 
- Form submitted successfully ✔️ 
- Github flow used ✔️ 
- Professional Readme ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -250,5 +250,8 @@ <h1>About <br />Myself</h1>
        
       </ul>
```
Comment (Talha-Hanif5666) [inline index.html]: Nice work though, but i think it is better not to leave any blank lines in your code where unnecessary. Doing this your code looks your code unclear.
Kindly please remove that blank line. Thank you

**Comment by leehaney254** [comment]
Nice work. Professional and descriptive commits. :smile:

**Review by Rana-Mudassir — COMMENTED** [review]
Everything is great. The code is clean. It would be better if you add a new commit when you complete a section or some feature in your code. Also, add some comments in your code about the sections and other features for better understanding.

**Review by Talha-Hanif5666 — COMMENTED** [review]
**Hello @Lucash2022**
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4b7942657366386b4e5436733838534567622f67697068792e676966


**Highlights**
Great job setting up the correct :heavy_check_mark: 
Professional documentation :heavy_check_mark: 
Correct workflow :heavy_check_mark: 
Meaningful commit messages :heavy_check_mark: 

There are some issues that you still need to work on to go to the next project but you are almost there!

#Changes Required recycle
Nice work and you did it so well +1
I have leave a comment that want changes, Kindly please see my comment and change the code according to it. Thank You

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! :partying_face: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: index.html (hunk 1 above)
Comment (Lucas-Erkana) [reply index.html]: Resolved this.

**Pull request merged**